resources/views/partials/analytics.blade.php

```diff
@@ -18,3 +18,19 @@
   _gs('{{ $gosquaredTracking }}');
 </script>
 @endif
+@if($piwikTracking = Setting::get('app_analytics_pi_url'))
```

Comment:
Ok, so as well as setting a default value within the dashboard, you should pass through a default value in the `Setting::get()` method:

```php
Setting::get('app_analytics_pi_siteid', 1)
```

This means that if they remove the default value, it'll still default.